drivers/serial/uart_nrfx_uart.c

```diff
@@ -407,24 +454,41 @@ static int uart_nrfx_init(struct device *dev)
 }
 
 static const struct uart_driver_api uart_nrfx_uart_driver_api = {
-	.poll_in          = uart_nrfx_poll_in,          /** Console I/O function */
-	.poll_out         = uart_nrfx_poll_out,         /** Console I/O function */
-	.err_check        = uart_nrfx_err_check,        /** Console I/O function */
+	/** Console I/O function */
```

Comment:
What was wrong with the previous layout?
Including such changes in the same commit obscures the actual modifications that the commit was intended to introduce.

Reply:
I was afraid that I have edited a line which exceed 80 characters span. I will remove these comments.